dataverse-webapp/src/main/java/edu/harvard/iq/dataverse/engine/command/impl/ReturnDatasetToAuthorCommand.java

```diff
@@ -52,12 +53,16 @@ public Dataset execute(CommandContext ctxt)  {
             Then remove reviewers from the autors list
             Finally send a notification to the remaining (non-reviewing) authors - Hey! your dataset was rejected.
         */
+        AuthenticatedUser requestor = getUser().isAuthenticated() ? (AuthenticatedUser) getUser() : null;
```

Comment:
This can be already an option there is no reason to return null. Just return Option.none as second result.